### ISSUE TITLE
This repository can be used for production?

### ISSUE THREAD

**Issue by vallemar** [rigor789/nativescript-vue-next#384]
Hi, I want to know if this repository can be used for a project in production.  I am developing a page with vue composition api at the same time I developed the app and I have to pass the same logic between the composition API and the option API. I would like to be able to develop a single codebase with the composition api.

**Comment by rigor789** [comment]
No, definitely not battle tested yet. It's fine to tinker with & report issues, but I would not use it in production just yet - it's not even in alpha yet, just a dev release to try it.

**Comment by vallemar** [comment]
Ok, I tried the vue 2 plugin for composition api and it seemed to work fine. Do you know if there are any limitations?

**Comment by pietdevries94** [comment]
> Ok, I tried the vue 2 plugin for composition api and it seemed to work fine. Do you know if there are any limitations?

Personally I found out that provide and inject doesn't work between frames

**Comment by Fabiyo-90** [comment]
I am going to create a complete application with nativescript-vue@3.0 then report the bugs and malfunctions, this can help @rigor789, also become a contributor by helping how to solve different bugs.
Big thank you to you, @rigor789 , for this gift including nativescript-vue, I will take my trouble patiently by undergoing this torture hoping that nativescript-vue@3.0 will be launched at the same time as nativescript@8.0

**Comment by rigor789** [comment]
> nativescript-vue@3.0 will be launched at the same time as nativescript@8.0

Definitely not going to happen, but I do hope I can flesh out the remaining bits soon after.

**Comment by neeravp** [comment]
@rigor789 Have you planned any tentative timeline for when it (complete support for Vue 3) would reach beta release - for eg Q1 2022. Just want to get an idea when will NativeScript-Vue be production ready with complete Vue 3 support.
BTW thank you all your efforts - making and managing Nativescript-Vue

**Comment by radudiaconu0** [comment]
@rigor789 yeah i really wait this release for a long time....

**Comment by superbiche** [comment]
@rigor789 Hey, any way some people could help? We're a small team quite fluent in Vue 3, honestly I'm not a NativeScript expert at all, but the perspective of being able to use it with Vue 3 makes learning it intensively quite worth the time. Let me know if we could help in any way.

**Comment by rigor789** [comment]
@superbiche Hey, I think the main thing missing right now is a bit of documentation to get started, and then some battle testing - building out small/medium demos, POC apps to find bugs, issues etc. - which then would need to be addressed.

From my side, I really need to:
1. write a better README how to use
2. publish an alpha
3. publish an alpha of the template

If you haven't already, definitely join the NativeScript discord https://nativescript.org/discord

I'm usually around there, so would be easier to communicate.

I'm currently working towards the 8.2 release of NativeScript, after which I do want to start building an app with vue3 to find issues. There are definitely missing pieces, those I tend to add gradually. The main thing is to wrap up the ListView/TemplateBag (v-template) handling, as I believe that was a quite nice api and would like to continue with it in vue3...

**Comment by ennioVisco** [comment]
> @superbiche Hey, I think the main thing missing right now is a bit of documentation to get started, and then some battle testing - building out small/medium demos, POC apps to find bugs, issues etc. - which then would need to be addressed.
> 
> From my side, I really need to:
> 
> 1. write a better README how to use
> 2. publish an alpha
> 3. publish an alpha of the template
> 
> If you haven't already, definitely join the NativeScript discord https://nativescript.org/discord
> 
> I'm usually around there, so would be easier to communicate.
> 
> I'm currently working towards the 8.2 release of NativeScript, after which I do want to start building an app with vue3 to find issues. There are definitely missing pieces, those I tend to add gradually. The main thing is to wrap up the ListView/TemplateBag (v-template) handling, as I believe that was a quite nice api and would like to continue with it in vue3...

Hey there, 8.2 is out, any plans for the vue3 app? I'm definitely willing to help in battle-test it!
